packages/ui/src/styles/globals.css

```
@@ -201,14 +201,14 @@
  * ones; without the var() the declaration silently never reaches the browser. */
 @utility glass-surface {
     background-color: var(--glass-surface);
-    --glass-blur: blur(16px) saturate(150%);
+    --glass-blur: blur(8px) saturate(120%);
     -webkit-backdrop-filter: var(--glass-blur);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)